nim/mike/config.yaml

```
@@ -1,6 +1,6 @@
 framework:
   github: ire4ever1190/mike
-  version: 0.5
+  version: 1.1.0
```

I prefer to use `X.Y` format
```suggestion
  version: 1.1
```